### ISSUE TITLE
Modalizer should allow focus to be completely trapped

### ISSUE THREAD

**Issue by ling1726** [microsoft/tabster#122]
Currently modalizer focus trap, allows the focus to escape the document into the window. This behaviour should be configurable and allow the focus trap to be circular within the modalizer and never escape
